### DISCUSSION TITLE
Add an option: Select only the Name column when selecting

### DISCUSSION THREAD

**Discussion by vagra** [doublecmd/doublecmd#1548] [Suggestions]
Currently, when we select files, the entire row is selected, including the Name column, Extension column, Size column, Date column, and Attributes column. Selecting the entire row leaves little blank space in the file window.

However, the blank space is quite useful. It allows double-clicking to go back one level and right-clicking to bring up the context menu. At present, with the entire row being selected, these two operations feel rather constrained.

**Comment by vagra** [comment]
like this screenshot:
![20240401_090309](https://github.com/doublecmd/doublecmd/assets/1576687/8845172d-fc94-46ea-9b7b-9cf8748da285)

**Comment by ags1234** [comment]
I don't know if you know this, but regardless of the selected files, you can go back one level by pressing Alt + left_arrow.

**Reply by vagra** [reply]
As a Windows user, I'm not very familiar with the shortcut system of Total Commander. However, I have indeed discovered in these past two days that I can use the arrow keys to directly return to the previous level or select a folder to enter the next level.

**Reply by ags1234** [reply]
If you use Show - Full, you can use the arrow keys to enter directory or go back on directory.

If you use Show - Brief, like I do, for example, you see the objects in columns. And you can navigate roughly with the left/right arrows, to jump on the column you need. Then up/down to position on the item you want.
To go up one level, Home then Enter.
